nginx/default.dev.conf

```diff
@@ -10,8 +10,6 @@ server {
 
         proxy_pass http://backend:4000;
         proxy_redirect off;
-
-        proxy_cookie_path / "/; secure; HttpOnly; SameSite=strict";
```

Comment:
removed because setting cookies to `secure` in development will cause them not to be sent to Infisical (via http)